diff --git a/setup.sh b/setup.sh
index deecbe1..0957fc9 100755
--- a/setup.sh
+++ b/setup.sh
@@ -291,12 +291,7 @@ installScript()
 installAll()
 {
 	for script in $SCRIPTS; do
-		local action=$(promptUser "Install ${script}? (Y/n)" "" "yYnN" "y")
-		case "$action" in
-			""|y|Y )	installScript install "${script}"
-					;;
-			*)		echo ""
-		esac
+		installScript install "${script}"
 	done
 }
 
@@ -416,21 +411,9 @@ installer()
 	if [ "$#" -eq 0 ]; then
 		printHeader "Installation wizard for synth-shell"
 
-		local action=$(promptUser "Would you like to install or uninstall synth-shell?" "[i] install / [u] uninstall. Default i" "iIuU" "i")
-		case "$action" in
-			""|i|I )	local install_option="install" ;;
-			u|U )		local install_option="uninstall" ;;
-			*)		printError "Invalid option"; exit 1
-		esac
-
-
-		local action=$(promptUser "Would you like to install it for your current user only (recommended),\n\tor system wide (requires elevated privileges)?" "[u] current user only / [s] system wide install. Default u" "uUsS" "u")
-		case "$action" in
-			""|u|U )	installerUser $install_option ;;
-			s|S )		sudo bash -c "bash $0 $install_option" ;;
-			*)		printError "Invalid option"; exit 1
-		esac
-
+		local install_option="install"
+		installerUser $install_option
+		sudo bash -c "bash $0 $install_option"
 	else
 		installerSystem $1
 
